sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c

```diff
@@ -395,12 +361,11 @@ void calc_fast9_lukas_kanade(struct opticflow_t *opticflow, struct opticflow_sta
 
   // Estimate size divergence:
   if (SIZE_DIV) {
-    n_samples = 100;
-    size_divergence = get_size_divergence(vectors, result->tracked_cnt, n_samples);
-    result->div_size = size_divergence;
+    result->div_size = get_size_divergence(vectors, result->tracked_cnt, 100);// * result->fps;
```

Comment:
I don't like the 100 without having a variable name saying what it is. Perhaps make n_samples a parameter, or define it in the top of the file if you don't like it here.